clippy.toml

```diff
@@ -0,0 +1,5 @@
+# The default clippy value for this is 8 bytes, which is chosen to improve performance on 32-bit.
+# Given that druid is being designed for the future and already even mobile phones have 64-bit CPUs,
```

Comment:
> and already even mobile phones have 64-bit CPUs

Is it correct to place `already` there?

> and even mobile phones already have 64-bit CPUs

feels better to me.

Reply:
Both are fine and you could even put it at the end. Check out this [Cambridge dictionary page](https://dictionary.cambridge.org/grammar/british-grammar/already) about it.